lib/facter/sensu_facts.rb

```diff
@@ -19,7 +19,7 @@ def self.get_version_info(cmd)
       build = $1
     end
     built = nil
-    if output =~ /built '(.*)'$/
+    if output =~ /built (?:')?([^']+)(?:')?$/
```

Comment:
Could you please add a comment that describes this regex

Reply:
Added comment about matching optional single quotes.